apps/table-plugin/src/app/app.component.ts

```diff
@@ -77,7 +77,7 @@ export class AppComponent {
           ?.split(/\r?\n|\r|\n/g)
           .map((it) => it.trim())
           .filter((it) => it !== '')
-          .map((it) => it.split(','));
+          .map((it) => it.split(',').map((it) => (!it ? ' ' : it.trim())));
```

Comment:
Following previous operations structure and just for visual clarity, shouldn't this last `map()` operation go in a newline?